.circleci/config.yml

```diff
@@ -414,6 +432,16 @@ workflows:
                 - /^dockerpush.*/
             tags:
               ignore: /.*/
+
+      - deploy-fxa-ci-image:
+          filters:
+            branches:
+              only:
+                - main
+                - FXA-6260
```

Comment:
This still needed?

Reply:
Nope. That was just for testing. Should have called this out myself. I will remove it.